v1/formats/wsdc_prep.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name>WSDC with Prep Times</name>
```

Comment:
```suggestion
  <name>World Schools with Prep Times</name>
  <short-name>WSDC with Prep</short-name>
```
For consistency with existing WSDC format names. I'm generally trying to avoid acronyms in the main name. `<name>` appears in the formats list, while `<short-name>` if provided appears in the title bar (falls back to `<name>` if not present)